app/components/message/message.css

```diff
@@ -11,3 +11,19 @@
   color: var(--color-text-light);
   font-size: var(--font-size-s);
 }
+
+.Message-photos {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
```

Comment:
Why do you need a flexbox if all you do is displaying several images below each other?